src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,107 @@
+.page {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $main-color;
```

Comment:
Ensure that the SCSS variable `$main-color` is defined in your stylesheets. Without a definition, this will cause an error during compilation.